[Rejected] Fix damaged note block

skycommand opened the pull request:
**This proposal has been rejected** with the following note from the author:

> Unfortunately, ]markdown formatting is not allowed in HTML tables. I adjusted the text formatting accordingly.

Original text:
>The PR intends to fix the damaged state of the "Version Helper functions" page on Microsoft Docs. Perhaps a couple of images best show what it tries to accomplish.
>
>Currently, the page looks like this:
>![How it is](https://user-images.githubusercontent.com/17097175/111882801-c8323780-89cc-11eb-8afb-e4766b392d1a.png)
>
>I believe the author intended it to look like this:
>![How it must be](https://user-images.githubusercontent.com/17097175/111882775-a9cc3c00-89cc-11eb-8d85-cbf1932b1179.png)

skycommand committed:
Fix damaged note block

PRMerger7 commented:
@skycommand : Thanks for your contribution! The author(s) have been notified to review your proposed change.

mcleanbyron committed:
Edited change

Unfortunately, ]markdown formatting is not allowed in HTML tables. I adjusted the text formatting accordingly.

PRMerger6 commented:
@mcleanbyron : Thanks for your contribution! The author(s) have been notified to review your proposed change.
